src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -207,7 +207,7 @@ private void setUpAlertsTriggers(List<Trigger> triggers) {
         List<Trigger> filteredTriggers = new ArrayList<>();
 
         for (Trigger trigger : triggers) {
-            if (trigger.getId().startsWith(resource.getId())) {
+            if (trigger.getId().endsWith(resource.getId())) {
```

Comment:
Well, if you want to do that in a separate PR it's fine, just to note that probably the trigger.getId() might not follow the resource.getId() in some case as it is not strong linked.
So using triggert.getTags().get("resourceId") might be better.
But anyway, if this change creates issues you can do it later in a separate PR.
It is good to have a PR per topic and not mix them for easy review.